package.json

```diff
@@ -118,7 +118,7 @@
     "postmarkdown": "prettier --write \"starters/**/*.md\"",
     "plop": "plop",
     "prebootstrap": "yarn",
-    "prettier": "prettier \"**/*.{md,css,scss,yaml,yml}\"",
+    "prettier": "prettier .",
```

Comment:
Can we rever this? Why changing this in the first place?

Reply:
I think linting fails because of this at the moment

Reply:
Because Prettier can be run on each possible file type.
If it's not supported, it won't crash.